proto/net.proto

```diff
@@ -65,18 +65,55 @@ message Weights {
     optional SEunit se = 3;
   }
 
+  message MHA {
+    optional Layer q_w = 1;
+    optional Layer q_b = 2;
+    optional Layer k_w = 3;
+    optional Layer k_b = 4;
+    optional Layer v_w = 5;
+    optional Layer v_b = 6;
+    optional Layer dense_w = 7;
+    optional Layer dense_b = 8;
+  }
+
+  message FFN {
+    optional Layer dense1_w = 1;
+    optional Layer dense1_b = 2;
+    optional Layer dense2_w = 3;
+    optional Layer dense2_b = 4;
+  }
+
+  message EncoderLayer {
+    optional MHA mha = 1;
+    optional Layer ln1_gammas = 2;
+    optional Layer ln1_betas = 3;
+    optional FFN ffn = 4;
+    optional Layer ln2_gammas = 5;
+    optional Layer ln2_betas = 6;
+  }
+
   // Input convnet.
   optional ConvBlock input = 1;
 
   // Residual tower.
   repeated Residual residual = 2;
 
+  // Policy encoder stack
+  repeated EncoderLayer encoder = 21;
```

Comment:
If we're going to potentially have a follow up PR for 'end of residual tower encoders' I think we should rename these both to have pol_ prefix - yes we'll have to update the other PRs, but better to do it now.